studies/_testing/generic_imitation_like_example1.py

```diff
@@ -109,6 +124,9 @@
                 else p_samesoc if i.social_system == j.social_system \
                 else p_other):
             culture.acquaintance_network.add_edge(i, j)
+for s1 in social_systems:
+    for s2 in social_systems:
+        culture.acquaintance_network.add_edge(s1, s2)
```

Comment:
isn't `acquaintance_network` between individuals rather than between social systems? what does it mean that two social systems are acquainted?

Reply:
hm, i can't remember what i must have thought here. but as this is only a test, maybe it's unimportant. but the metadata model says acquaintance_network is between Individuals, so maybe we should remove these lines here.